app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors
+
+    @ordered_actors = @movie.order_actors
+    @average_age_of_cast = @movie.average_age
```

Comment:
This is fine, but you can also access `@movie.order_actors` and any other model methods from the view - you don't have to send them all down as different instance variables from the controller. That way, you are keeping the controller action "lightweight" and easy to read and maintain.